src/main/java/org/candlepin/util/OIDUtil.java

```diff
@@ -99,14 +105,15 @@ private OIDUtil() {
 
     static {
         // load top level namespaces
-        TOPLEVEL_NAMESPACES.put(PRODUCT_CERT_NAMESPACE_KEY, "1");
-        TOPLEVEL_NAMESPACES.put(CHANNEL_FAMILY_NAMESPACE_KEY, "2");
-        TOPLEVEL_NAMESPACES.put(ROLE_ENT_NAMESPACE_KEY, "3");
-        TOPLEVEL_NAMESPACES.put(ORDER_NAMESPACE_KEY, "4");
-        TOPLEVEL_NAMESPACES.put(SYSTEM_NAMESPACE_KEY, "5");
-        TOPLEVEL_NAMESPACES.put(ENTITLEMENT_VERSION_KEY, "6");
-        TOPLEVEL_NAMESPACES.put(ENTITLEMENT_DATA_KEY, "7");
-        TOPLEVEL_NAMESPACES.put(ENTITLEMENT_TYPE_KEY, "8");
+        TOPLEVEL_NAMESPACES.put(Namespace.PRODUCT_CERTIFICATE, "1");
+        TOPLEVEL_NAMESPACES.put(Namespace.CHANNEL_FAMILY, "2");
+        TOPLEVEL_NAMESPACES.put(Namespace.ROLE_ENTITLEMENT, "3");
+        TOPLEVEL_NAMESPACES.put(Namespace.ORDER, "4");
+        TOPLEVEL_NAMESPACES.put(Namespace.SYSTEM, "5");
+        TOPLEVEL_NAMESPACES.put(Namespace.ENTITLEMENT_VERSION, "6");
+        TOPLEVEL_NAMESPACES.put(Namespace.ENTITLEMENT_DATA, "7");
+        TOPLEVEL_NAMESPACES.put(Namespace.ENTITLEMENT_TYPE, "8");
+        TOPLEVEL_NAMESPACES.put(Namespace.PRODUCT, "9");
```

Comment:
This should probably be named something closer to "entitlement namespace" or "product namespace" instead of "product"

Reply:
Done.